nodebuilder/testing.go

```diff
@@ -46,7 +48,9 @@ func TestNodeWithConfig(t *testing.T, tp node.Type, cfg *Config, opts ...fx.Opti
 	require.NoError(t, err)
 	cfg.Core.IP = ip
 	cfg.Core.RPCPort = port
-	cfg.RPC.Port = "26655"
+	freePort, err := tennet.GetFreePort()
```

Comment:
As discussed, we go with 0 rather than this. #1322

Reply:
done